goimports/action.yaml

```diff
@@ -24,7 +28,7 @@ runs:
       for dir in $(find . -name go.mod -maxdepth 3 -exec dirname {} \; ); do
         echo "::group:: $dir"
         pushd $dir
-        goimports -w \
+        goimports -local="${{ inputs.local }}" -w \
```

Comment:
cc @n3wscott 

Reading about this option a bit, I think it is something you might be interested in for us downstream, do you want to TAL at this and review?

Reply:
@mattmoor for PR details:
I planned use this change to the https://github.com/sigstore/cosign/blob/20c8c88f3157a16e15be07f926aa3c2dba9bbfec/.github/workflows/style.yaml#L42.
Also planned send PR to sigstore/cosign about fix import section with goimports with `-local=github.com/sigstore`.